### PR TITLE
Feature/logo

### PR DESCRIPTION
**Pull request by DeFidelity**
Completed logo addition and changed the web image in favicon to favicon-sized logo image.

Logo view:
Screenshot 2023-02-22 174254

Favicon view:
Screenshot 2023-02-22 174358

### CODE REVIEW

**Comment by turian** [comment]
Why have so many files changed?

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# flyctl launch added from .gitignore
+**/*.toml
+# Byte-compiled / optimized / DLL files
```
Comment (turian) [inline .dockerignore:3]: should not get checked in. fly copies it automatically. don't add

File: fly-staging.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+app = "audio-discrimination-crowdsource-thisfidelity-staging"
```
[the remark this answers is not in the document]
Comment (DeFidelity) [reply fly-staging.toml]: All resolved

File: Dockerfile [hunk 3]
```diff
@@ -1,25 +0,0 @@
-ARG PYTHON_VERSION=3.10-slim-buster
-
-FROM python:${PYTHON_VERSION}
```
Comment (turian) [inline Dockerfile:3]: Deleted why?

File: delete-all-fly-apps.py [hunk 4]
```diff
@@ -1,32 +0,0 @@
-#!/usr/bin/env python3
-
-import json
-import os
```
Comment (turian) [inline delete-all-fly-apps.py:4]: Deleted why?

File: fly-staging.toml.tmpl [hunk 5]
```diff
@@ -1,45 +0,0 @@
-app = "audio-discrimination-crowdsource-HANDLE-staging"
-kill_signal = "SIGINT"
-kill_timeout = 5
-processes = []
-
-[build]
```
Comment (turian) [inline fly-staging.toml.tmpl:6]: Deleted why?

**Review by turian — APPROVED** [review]

**Pull request merged**